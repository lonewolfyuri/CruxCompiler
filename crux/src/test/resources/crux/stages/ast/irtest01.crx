
// test printBool

func main() : void {
  ::printBool(true);
  ::printBool(false);
}
