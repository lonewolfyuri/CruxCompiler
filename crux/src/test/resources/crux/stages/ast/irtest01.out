ast.DeclarationList(4,0)
  ast.FunctionDefinition(4,0)[Symbol(main), []]
    ast.StatementList(5,2)
      ast.Call(5,2)[Symbol(printBool)]
        ast.LiteralBool(5,14)[TRUE]
      ast.Call(6,2)[Symbol(printBool)]
        ast.LiteralBool(6,14)[FALSE]
